CHANGELOG.md

```diff
@@ -19,6 +19,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security
 
+
+## 0.1.3
+
+Released on October 11th, 2022.
+
+### Added
+
+- Added `cwd` keyword argument in `shell_run_command` - [#41](https://github.com/PrefectHQ/prefect-shell/pull/41)
```

Comment:
Can you move this changelog entry to Unreleased -> Added? This helps us keep track of changes that are merged into main, but not released yet. We'll move this entry under a version heading right before it gets released along with any other unreleased changes.

Reply:
Totally. Done that now. Wasn't sure exactly about how that would all work regarding versioning and release timing, but that makes so much more sense now.